src/wallet/wallet-connection.ts

```diff
@@ -0,0 +1,56 @@
+import { Account, SignAndSendTransactionOptions } from '../account';
+import { Connection } from '../connection';
+import { Near } from '../near';
+import { FinalExecutionOutcome } from '../providers';
+import { Wallet } from './interface';
+
+export class WalletConnection {
+    /** @hidden */
+    _wallet: Wallet;
+
+    /** @hidden */
+    _near: Near;
+
+    /**
+    * @param {Near} _near Near object
+    * @param {Walle} _wallet Wallet object
+    */
+    constructor(near: Near, wallet: Wallet) {
```

Comment:
why not just `private near: Near, private wallet: Wallet`?, they should be hidden by default.

Reply:
And why `Near` and not `connection`?